orchestration/src/main/java/com/sap/ai/sdk/orchestration/AssistantMessage.java

```diff
@@ -0,0 +1,25 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.sap.ai.sdk.orchestration.client.model.ChatMessage;
+import javax.annotation.Nonnull;
+
+/**
+ * Represents a chat message as 'assistant' to the orchestration service.
+ *
+ * @param content
+ */
+public record AssistantMessage(@Nonnull String content) implements Message {
```

Comment:
I think I'd make this a class, because the type of content will change in the future. Records are not as easily extensible and make the getters public, which is an issue if content needs to be a complex type